.github/workflows/release.yml

```diff
@@ -0,0 +1,19 @@
+---
+name: release
+
+on:
+  push:
+    tags:
+      - v*.*.*
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Release Gem
+        uses: cadwallion/publish-rubygems-action@master
+        env:
+          RUBYGEMS_API_KEY: ${{ secrets.RUBYGEMS_API_KEY }}
+          RELEASE_COMMAND: rake build
```

Comment:
`rake build` will not release. It will just "build". To release you need to use the `rake release` task.

See the differences between `build`:

https://github.com/rubygems/rubygems/blob/e360e8a58006055dc1f0db7b26c8ead4d91a338e/bundler/lib/bundler/gem_helper.rb#L45-L48

and `release`:

https://github.com/rubygems/rubygems/blob/e360e8a58006055dc1f0db7b26c8ead4d91a338e/bundler/lib/bundler/gem_helper.rb#L65-L69